deploy/kubeturbo-operator/certified-operator-automation-1.py

```diff
@@ -0,0 +1,178 @@
+import os
+import shutil
+from github import Github
+import git
+import re
+import docker
+
+#icr parameters
+icr_url = 'icr.io'
+icr_namespace = 'cpopen'
+#Enter the local repo path to clone repository to modify files
+local_repo_path = os.getenv('LOCAL_REPO_PATH')
+if local_repo_path is None:
+    local_repo_path = input('Enter repository local path: ')
+#PR parameters
+base_repo_name = 'certified-operators'
+base_repo_owner = 'redhat-openshift-ecosystem'
+origin_repo_name = 'certified-operators'
+origin_repo_owner = 'turbodeploy'
+#Git access token
+access_token = os.getenv('ACCESS_TOKEN')
+if access_token is None:
+    access_token = input('Enter turbodeploy access token: ')
```

Comment:
I recommend to use getpass to get any secret from user.
https://docs.python.org/3/library/getpass.html